### ISSUE TITLE
Invalid format error

### ISSUE THREAD

**Issue by masonfox** [maxswa/osrs-json-hiscores#51]
Receiving an invalid format error:
```
C:\Users\mason\git\osrs-buddy-bot\node_modules\osrs-json-hiscores\lib\hiscores.js:109
        throw Error(utils_1.INVALID_FORMAT_ERROR);
              ^
Error: Invalid hiscores format
```
It looks as though the hiscore csv has changed? Here is a link to [my account](https://secure.runescape.com/m=hiscore_oldschool/index_lite.ws?player=Xofy) via the API.

**Comment by maxswa** [comment]
This will be solved by this PR: https://github.com/maxswa/osrs-json-hiscores/pull/52

**Comment by maxswa** [comment]
The latest version fixes this issue by adding support for "Rifts closed" `osrs-json-hiscores@2.8.0`
